README.md

```diff
@@ -47,6 +47,43 @@ environment variables can be used to determine environment).  For example in
     production:
       "foo,bar,baz": 4
 
+### YAML file config [UPDATE]
```

Comment:
Also, since you chose to create a separate section for Skroutz-oriented configs I'd suggest to use a more descriptive tag than `[UPDATE]`, something like `[skroutz]` or something similar.

Reply:
Sure `skroutz` sounds ok

Reply:
Fixed!